skopt/utils.py

```diff
@@ -292,3 +295,44 @@ def cook_estimator(base_estimator, space=None, **kwargs):
 
     base_estimator.set_params(**kwargs)
     return base_estimator
+
+def transform_gpdims(dimensions):
```

Comment:
no you are right. I just thought that abstracting away the logic would be a good idea, in case we would like to make some changes in the future.